File: Semestr 1/TablicaChar/Python/kod_ASCII.py
'''
autor: Paweł Sadowski
Program wypisuje wartrosc ASCII dla liczb calkowitych z zadanego zakresu
'''
n_p = 0
n_k = 0

n_p = int(input(""))
n_k = int(input(""))

for i in range(n_p, n_k + 1):
    print(i, ' - ', chr(i))

File: Semestr 1/TablicaChar/C++/Kod_ASCII.cpp
/*autor: Pawel Sadowski*/
/* Program wypisuje wartrosc ASCII dla liczb calkowitych z zadanego zakresu */
#include <iostream>
using namespace std;

int main()
{
    int n_p,n_k;

    cin >> n_p >> n_k;

    for(int i=n_p; i <= n_k; i++)
    {
        cout << i << " - " << char(i);
        cout << endl;
    }
    return 0;
}
